MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java

```diff
@@ -72,7 +75,7 @@
  * of {@link ExoPlayer} version and make sure that all events are passed to {@link MuxStats} in
  * the correct order and with appropriate statistics.
  */
-public class MuxBaseExoPlayer extends EventBus implements IPlayerListener {
+public abstract class MuxBaseExoPlayer extends EventBus implements IPlayerListener {
```

Comment:
Making this class `abstract` feels like the right thing to do, given that it's always extended by Exoplayer-version-specific subclasses. 👍